src/Routers/routes/ShiftingRoutes.tsx

```diff
@@ -7,12 +7,14 @@ import BoardView from "@/components/Shifting/ShiftingBoard";
 import ListView from "@/components/Shifting/ShiftingList";
 
 import { AppRoutes } from "@/Routers/AppRouter";
-
-const getDefaultView = () =>
-  localStorage.getItem("defaultShiftView") === "list" ? "list" : "board";
+import { getDefaultView } from "@/Utils/viewStorageUtils";
 
 const ShiftingRoutes: AppRoutes = {
-  "/shifting": () => <Redirect to={`/shifting/${getDefaultView()}`} />,
+  "/shifting": () => (
+    <Redirect
+      to={`/shifting/${getDefaultView("defaultShiftingView", "board")}`}
+    />
+  ),
   "/shifting/board": () => <BoardView />,
   "/shifting/list": () => <ListView />,
```

Comment:
yup, let's remove these routes too and instead we could do:

(which uses that hook behind the scenes)
```
"/shifting": () => <View 
  name="shifting"
  board={BoardView}
  list={ListView}
  />
```